src/createActions.ts

```diff
@@ -1,23 +1,24 @@
-import { IActions, IActionTypes } from './typings';
+import { IActions, IActionTypes, IActionsTypes } from './typings';
 import createAction from './createAction';
 
 /**
  * generate multiple Flux Standard Action creators
  * @param {IActionTypes} actionTypes
  * @returns {IActions<any>}
```

Comment:
Please modify the function signature at the same time.

Reply:
😂Ohhhh, It's my mistake. I forgot it at all...